low-level/files/opt/seagate/sspl/conf/sspl.config.tmpl.1-node.sample

```diff
@@ -32,7 +32,7 @@ server_node:
 storage_enclosure:
   enc30512e5ae6df9f1ea02327bab45e499d:
     controller:
-      password: Admin123
+      secret: Admin123
```

Comment:
sample value should follow encryption too

Reply:
Done.